src/components/__tests__/__snapshots__/markdown.test.js.snap

```diff
@@ -54,7 +54,7 @@ exports[`Markdown renders devDynamicBlogs is true 1`] = `
   />
   <DisplayWork
     link="blog"
-    prefix="📝 I regulary write articles on"
+    prefix="📝 I regularly  write articles on"
```

Comment:
@aravindvnair99 It looks like you have added a double space after `regularly` in every correction.

Reply:
@harshcut Ah yes, thanks for noticing. I hadn't realised that. I've fixed it now.